### ISSUE TITLE
Change uploads location

### ISSUE THREAD

**Issue by smutax** [chibisafe/chibisafe#676]
Hi. Followed installation with Docker and Caddy.  Couldn't get the path for uploads to point to an external hdd mounted to DietPi on Raspberry pi. How should things be changed for all the uploads to go to that external hdd? Any help appreciated. Thanks.

**Comment by jamesbt365** [comment]
I don't believe chibisafe currently supports this, but you can just create a symlink.

**Comment by TheDogHusky** [comment]
If you mount your HDD, on linux, it should be somewhere in /run/media and just work as path for the uploads.
I already ran a chibisafe on a computer with an SD card containing the uploads folder.

**Comment by smutax** [comment]
> I don't believe chibisafe currently supports this, but you can just create a symlink.

Thanks for clarification. So i need a script on hdd creating symlinks and placing them in chibisafe uploads? Will it work ,as chibisafe renaming uploaded files giving them different names?

**Comment by Pitu** [comment]
> > I don't believe chibisafe currently supports this, but you can just create a symlink.
> 
> Thanks for clarification. So i need a script on hdd creating symlinks and placing them in chibisafe uploads? Will it work ,as chibisafe renaming uploaded files giving them different names?

No, you create just 1 symlink linking the uploads folder from chibisafe to another folder in your hdd

**Comment by smutax** [comment]
> > > I don't believe chibisafe currently supports this, but you can just create a symlink.
> > 
> > 
> > Thanks for clarification. So i need a script on hdd creating symlinks and placing them in chibisafe uploads? Will it work ,as chibisafe renaming uploaded files giving them different names?
> 
> No, you create just 1 symlink linking the uploads folder from chibisafe to another folder in your hdd

So to do that i  need to stop chibisafe service, then run "ln -s /my_hdd/ /uploads and then restart chubisafe and that will make all the uploads to end up in my hdd without taking up space in /uploads on sd card? Thanks

**Comment by Pitu** [comment]
If you are on the chibisafe folder, the command would look like:
```bash
sudo ln -s /my_hdd ./uploads
```

This will make so that any file uploaded to chibisafe will end up in `/my_hdd`

**Comment by smutax** [comment]
> If you mount your HDD, on linux, it should be somewhere in /run/media and just work as path for the uploads. I already ran a chibisafe on a computer with an SD card containing the uploads folder.

So if it's in the /run/media will it appear in chibisafe without doing anything? Btw, just checked, there's no /media in /run with my hdd mounted. Hdd is showing in /mnt. Thanks.

**Comment by smutax** [comment]
> If you are on the chibisafe folder, the command would look like:
> 
> ```shell
> sudo ln -s /my_hdd ./uploads
> ```
> 
> This will make so that any file uploaded to chibisafe will end up in `/my_hdd`

Understood. I have to try that and report back. Thanks.

**Comment by smutax** [comment]
> If you are on the chibisafe folder, the command would look like:
> 
> ```shell
> sudo ln -s /my_hdd ./uploads
> ```
> 
> This will make so that any file uploaded to chibisafe will end up in `/my_hdd`

It didn't seem to work. It created "my_hdd" folder inside of uploads and linked it to actual /mnt/my_hdd, instead of linking uploads itself to my_hdd.

**Comment by Pitu** [comment]
Maybe I got the command wrong, you'll have to dig a bit what the correct command is for your case.
But the solution is solid, you symlink the uploads folder to another destination of your choosing

**Comment by jamesbt365** [comment]
The command is right for the most part, the only thing wrong is that the paths are backwards. its source then destination.
